src/TagField.php

```diff
@@ -402,6 +402,14 @@ protected function getListValues($values): array
             return $values->column($this->getTitleField());
         }
 
+        if ($values instanceof DataObject && $values->exists()) {
```

Comment:
We use `$this->getTitleField()` above, how come we're using `->ID` here?

Reply:
Since either IDs or titles work (https://github.com/silverstripe/silverstripe-tagfield/blob/2/src/TagField.php#L289) I opted for ID since it's unambiguous and most likely more efficient.
Should I change it to `return [$values->getField($this->getTitleField())];` ?

Reply:
Makes sense for it to be consistent.

Is there a scenario where $values won't have a column matching the title field? Do we need a fallback to the ID field, or is title field always safe?

Reply:
@emteknetnz I changed it to use the title-field instead of the ID.